src/Project/MvpSite/rendering/appsettings.json

```diff
@@ -3,7 +3,25 @@
     "InstanceUri": "http://mvp-cd",
     "LayoutServicePath": "/sitecore/api/layout/render/jss",
     "DefaultSiteName": "mvp-site",
-    "ApiKey": "{E2F3D43E-B1FD-495E-B4B1-84579892422A}"
+    "ApiKey": "{E2F3D43E-B1FD-495E-B4B1-84579892422A}",
+    "InstanceCMUri": "http://cm",
+    "Domain": "Sitecore",
+    "Username": "admin",
+    "Password": "b",
+    "MVPApplicationTemplateId": "5A6A420C-BE7E-4822-A887-33C83F5E9B0C",
+    "PersonTemplateId": "AD9C7837-8660-4360-BA2B-7ADDF4163685"
+  },
+  "Okta": {
+    "OktaDomain": "https://dev-45952888.okta.com",
+    "ClientId": "0oa1jhwqy63SWS4sG5d7",
+    "ClientSecret": "KrF9WOnKRCpYsQEjm-RPr9kJ-NjEHWpJBRyfpiUj",
+    "AuthorizationServerId": "default"
```

Comment:
Can we please remove these Okta details and change them to be populated by Environment variable.

Reply:
removed